package.json

```diff
@@ -33,7 +33,7 @@
     "start_win": "set DEBUG=raneto&&node example/server.js",
     "test": "npm run lint && mocha --reporter spec test/*.js",
     "gulp": "gulp",
-    "lint": "./node_modules/.bin/eslint bin/* app/**/*.js example/**/*.js test/*.js gulpfile.js",
+    "lint": "eslint bin/* app/**/*.js example/**/*.js test/*.js gulpfile.js",
```

Comment:
Reason for this? not everyone has eslint installed globally.
Will NPM handle this in such a case?

Reply:
Having some kind of dejavu: https://github.com/gilbitron/Raneto-Core/pull/15#issuecomment-233160749 :D

Just wanted to clarify that it wouldn't try a global installation, what `npm` does is to prepend all bin commands in your PATH before running the script, which means that local eslint will be picked up.

Reply:
Hi @ryanlelek, I can undo this change if you prefer. However I think this is the right way to set it up, it always worked like a charm for me in cross platform setups.

![image](https://user-images.githubusercontent.com/12942794/183648964-a7101d04-96c7-4c1d-b78f-0dc80d5dd2a3.png)

https://docs.npmjs.com/cli/v8/using-npm/scripts#path

Reply:
It makes sense, thank you for the clarification.
In `npm` world, it adds its `.bin/` to the search path. This may not(?) have always been the case which might be why I got in the habit.